info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -67,6 +70,15 @@
    *     of permission to access required libraries.
    */
   public WebCrawler() throws IOException {
+    this.datastore = DatastoreOptions.getDefaultInstance().getService();
+    this.relevancyChecker = new RelevancyChecker();
```

Comment:
```suggestion
    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
    this(datastore);
```
Can chain constructors to avoid duplication.